### PR TITLE
Added test configuration for ci workflow

### PR DESCRIPTION
**Pull request by brandonzhu09**
## Description
Created different configuration files to setup the database in the development and production phase of `evagram`. This allows for the configuration of workflow runs to test the database schema at each step of the lifecycle process.

## Issue(s) addressed

#6

### CODE REVIEW

File: src/evagram_input/input_tool/input_tool.py [hunk 1]
```diff
@@ -1,26 +1,36 @@
...
-    def __init__(self, owner: str, experiment: str, eva_directory: str):
+    def __init__(self, owner: str, experiment: str, eva_directory: str, test_local=False):
+        self.username = config.user
+        self.dbname = config.dbname
+        self.host = config.host
+
+        if test_local:
+            self.username = test_config.user
+            self.dbname = test_config.dbname
+            self.host = test_config.host
```
Comment (ghost) [inline src/evagram_input/input_tool/input_tool.py]: This is pretty much what I described in our last meeting, but here's an alternative solution:

Store `user/database/host` information in a `dbconfig` class defined in its own separate `dbconfig` module.  Move `config.py` and `test_config.py` into that module.  Initialize the object with the `test_local` boolean to pull in the proper db parameters.  Then have a `get` method that returns a dictionary like `{'user':self.user, 'database':self.database, 'host':self.host'}`.  In the session class where psycopg2 is called, we can do `connect(**dbparams)`.  Think that could work?  It seems like a cleaner way to organize concerns.

**Review by ghost — COMMENTED** [review]
See comment

File: src/evagram_input/input_tool/input_tool.py (hunk 1 above)
Comment (brandonzhu09) [reply src/evagram_input/input_tool/input_tool.py]: I believe that could work, I like the idea of moving the config files outside the session class. How will the `dbconfig` object interact with the session class? Will this also mean `dbconfig` is another dependency in evagram for testing?

Comment (brandonzhu09) [reply src/evagram_input/input_tool/input_tool.py]: I was thinking a bit more about the implementation, and I thought that I could replace the `test_local` boolean with an environment variable `APP_ENVIRONMENT` which I can set to the current state of the project (local). This way the `dbconfig` module can configure the database in multiple ways based on the value in `APP_ENVIRONMENT`. @tariqhamzeyssai Let me know if that sounds good with you.

**Comment by brandonzhu09** [comment]
Hi @tariqhamzeyssai, I believe this PR is ready to be merged but you can take a look and see the changes I have made. Thanks!

**Review by ghost — COMMENTED** [review]
Thanks Brandon!

**Review by asewnath — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
I thought I clicked the approve button last week

**Pull request merged**